ext/js/data/options-util.js

```diff
@@ -1132,6 +1133,36 @@ export class OptionsUtil {
         }
     }
 
+    /**
+     * - Change 'viewNote' action to 'viewNotes'
```

Comment:
Nitpick, but punctuation in the change notes.

Reply:
👍